static/rfcs/10/overview.msc

```diff
@@ -0,0 +1,41 @@
+# Sequence diagram for Waku v2 (WakuMessage, WakuData, Relay, Store, Filter)
+msc {
+  hscale="1",
+  wordwraparcs=true;
+
+  a [label="A [relay]"],
+  b [label="B [relay(topic1)]"],
+  c [label="C [relay(topic2)]"],
+  d [label="D [relay(topic1), store(topic1)]"],
+  e [label="E[relay, store]"],
+  f [label="F [relay, filter]"];
+
+#  b <-> c [label="connection"];
+#  a <-> b [label="connection"];
+#  b <-> d [label="connection"];
+#  d <-> e [label="connection"];
+#  c <-> d [label="connection"];
+
+  a rbox a [label="msg=WakuMessage(contentTopic1, data) [14/WAKU2-MESSAGE]"];
+  a note a [label="If version=1, encrypt data per [7/WAKU-DATA]"];
+
+#  b note b [label="relay: subscribe topic1"];
+#  c note c [label="relay: subscribe topic2"];
+#  d note d [label="relay: subscribe topic1"];
+
+  f => d [label="FilterRequest(topic1, contentTopic1) [12/WAKU2-FILTER]"];
+  d rbox d [label="filter: install for d [12/WAKU2-FILTER]"];
```

Comment:
What does `install for d` mean in this context? D should add F as subscriber to the requested filter at this point.

Reply:
Addressed